high_card.rb

```diff
@@ -1,5 +1,28 @@
 # Use these two arrays to generate a deck of cards.
+def cards
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+deck = []
+
+ranks.each_with_index do |r, i|
+    suits.each do |s|
+      deck.push({
+        ranks: r,
+        suit: s,
+      })
```

Comment:
Great use of each_with_index! Can you think of a way to save a card's 'value' to the hash you are pushing to the deck.
Maybe using the index: `i` you are passing into the block.